src/pages/MyFriendsPage.js

```diff
@@ -36,17 +42,18 @@ export default function MyFriendsPage({
     if (firstFriend > secondFriend) return 1;
     return 0;
   });
-
+  console.log(friendsOnlyOnceWithoutDetails);
```

Comment:
please check, whether this is indeed removed

Reply:
its already out -  thank you! 😊